src/components/MainHeader.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { NavLink, Link } from "react-router-dom";
+import styled from "styled-components";
+
+export default function MainHeader() {
+  return (
+    <MainHeaderContainer>
+      <nav>
+        <section>
+          <Link to="/">
+            <div className="logo">
+              <h2>anon-wall</h2>
+            </div>
+          </Link>
+        </section>
+        <ul className="mainHeader-link">
+          <li>
+            <NavLink
+              to="/counsels"
+              className={(navData) => (navData.isActive ? "active" : "")}
+            >
+              고민 담벼락
+            </NavLink>
+          </li>
+          <li>
+            <NavLink
+              to="/mypage"
+              className={(navData) => (navData.isActive ? "active" : "")}
+            >
```

Comment:
```suggestion
            <NavLink
              to="/counsels"
              className={({ isActive }) => (isActive ? "active" : "")}
            >
              고민 담벼락
            </NavLink>
          </li>
          <li>
            <NavLink
              to="/mypage"
              className={({ isActive }) => (isActive ? "active" : "")}
            >
```
구조할당분해로 통일하면 좋을 것 같아요~!

Reply:
구조분해할당으로 하면 더 깔끔하겠네요!